README.md

```diff
@@ -1,3 +1,161 @@
+
+## Quick start
+
+To run your Parasoft SOAtest and review test results on GitHub, you need to customize your GitHub workflow to include:
+- The action to run SOAtest.
+- The action to upload the transformed xUnit report to GitHub.
+
+### Prerequisites
+This action requires Parasoft SOAtest with a valid Parasoft license.
+
+We recommend that you run Parasoft SOAtest on a self-hosted rather than GitHub-hosted runner.
+
+### Adding the Run SOAtest Action to a GitHub Workflow
+Add the `Run SOAtest` action to your workflow to launch test suites with Parasoft SOAtest.
+
+At a minimum, the action requires the `soatestWorkspace` parameter to be configured to specify the path to a SOAtest workspace that determines the scope of analysis.
+
+### Uploading Analysis Results to GitHub and reviewing the results
+By default, the `Run SOAtest` action generates some reports in report directory. You can upload them by adding the upload-artifact action to your workflow.
+
+We recommend that setting `convertReportToXUnit` parameter to true to generate an xUnit report in XML format. You can then upload it by adding the `Publish Test Results` action in your workflow and review the results on GitHub. See [Publish Test Results](https://github.com/marketplace/actions/publish-test-results) for details.
+
+### Examples
+The following examples demonstrate simple workflows consisting of one job for project building. The examples assume that SOAtest is running on a self-hosted runner and the path to the `soatestcli` executable is available in the `PATH`.
+
+#### Run SOAtest with specified workspace
+
+```yaml
+
+# This is a basic workflow to help you get started with the Run SOAtest action.
+name: Run SOAtest with specified workspace
+
+on:
+  # Trigger the workflow on push or pull request events but only for the main branch.
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab.
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel.
+jobs:
+  build:
+    name: Run test suites with SOAtest
+    # Specifies the type of runner that the job will run on.
+    runs-on: self-hosted
+    # Steps represent a sequence of tasks that will be executed as part of the job.
+    steps:
+    # Checks out your repository under $GITHUB_WORKSPACE, so that your job can access it.
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    # Execute the tests with SOAtest.
+    - name: Run SOAtest
+      id: SOAtest
+      uses: parasoft/run-soatest-action@1.0.0
+      with:
+        #Specify a SOAtest workspace that determines the scope of analysis.
+        soatestWorkspace: soatest
+
+    # Upload the reports to GitHub as an artifact.
+    - name: Upload report artifact
+      uses: actions/upload-artifact@v2.2.3
```

Comment:
Why are we using lower version than the one used in Jtest action? If there is no special reason, please use higher version here and match pattern for `path` to be consistent with language tool YAML.